### PR TITLE
Fonts, colors, typography styles, block color palette

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Adds fonts from designs
- Adds colors from design
- Updates typography styling to match designs
- Adds block color palette using the colors added

## Why

For #62

## Testing/Questions

Features that this PR affects:

- Styling site wide

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?
- [ ] I tried to not add colors that were too similar to other ones. Am I missing any that should be added?
- [ ] Any other styles we should add to this PR?

Steps to test this PR:

1. View frontend and verify all fonts, colors, and typography matches the designs so far
2. Add a block and make sure the block color palette works as expected

### CODE REVIEW

**Comment by joshdarby** [comment]
Note that `@font-body` and `@font-header` are the same exact font. `@font-description` is different but is only used in 2 places in the design on the homepage. However, I've left `@font-body` and `@font-header` as two separate variables incase we ever do want to change one of them without the other.

File: wp-content/themes/ipbs-largo/inc/enqueue.php [hunk 1]
```diff
@@ -8,5 +8,14 @@ function ipbs_enqueue_styles(){
 		get_stylesheet_directory_uri() . '/css/child-style.css',
 		filemtime( get_stylesheet_directory() . '/css/child-style.css' )
 	);
+	// fonts
```
Comment (benlk) [inline wp-content/themes/ipbs-largo/inc/enqueue.php:12]: Not a comment on this change specifically, but in the `'largo-child-styles'` enqueue above, `filemtime( ... )` should be the 4th param, with the third as `null`.

File: wp-content/themes/ipbs-largo/less/variables.less [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+@color-heading: @darkblue;
+@color-body: @darkgrey;
+
+// copied from Largo for ease
+@navbarHeight: 40px;
+
+// Fonts
+@font-body: 'Inter', "Helvetica Neue", Helvetica, Arial, sans-serif;
+@font-description: 'source-sans-pro', "Helvetica Neue", Helvetica, Arial, sans-serif;
```
Comment (benlk) [inline wp-content/themes/ipbs-largo/less/variables.less:31]: Are we using this anywhere yet?

**Review by benlk — APPROVED** [review]

File: wp-content/themes/ipbs-largo/less/variables.less (hunk 2 above)
Comment (benlk) [reply wp-content/themes/ipbs-largo/less/variables.less:31]: Wait nevermind, just saw your comment.

File: wp-content/themes/ipbs-largo/inc/enqueue.php (hunk 1 above)
Comment (joshdarby) [reply wp-content/themes/ipbs-largo/inc/enqueue.php:12]: Updated in https://github.com/INN/umbrella-ipbs/pull/65/commits/fa1eb13ee0eb3f3f29bb521238621e2927c9cee0

**Pull request merged**